src/tests/not_duktape.rs

```diff
@@ -1,20 +0,0 @@
-use super::*;
```

Comment:
Nothing wrong with this `mod`. The only change needed is to put it under `render_test`.